### DISCUSSION TITLE
Cask not found error when trying to install from Brew website.

### DISCUSSION THREAD

**Discussion by aknight2015** [Homebrew/discussions#3408] [Casks]
I installed brew according the instructions. Zero errors. I want to install the below program:
fantasy-map-generator
Install command: brew install --cask fantasy-map-generator
I get an error:
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> New Formulae
cargo-bundle                             mle                                      uthash

Error: Cask 'fantasy-map-generator' is unavailable: No Cask with this name exists.
I got the install command via the search function on the site.

**Comment by SMillerDev** [comment]
Are you using Linux?

**Comment by aknight2015** [comment]
Yes, Debian (Bullseye).

**Reply by SMillerDev** [reply]
Casks are macOS only

**Reply by aknight2015** [reply]
Oh, well, that explains quite a bit.

**Comment by carlocab** [comment]
What is the output of `brew tap`, `brew config` and `brew doctor`?

**Reply by aknight2015** [reply]
brew tap:
homebrew/core
brew config:
HOMEBREW_VERSION: 3.5.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 88a5fac96752f17280e64f60926f58dff45990e0
Last commit: 3 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 4cd544aa50e284c4966e7c127901ad61fc606cc0
Core tap last commit: 64 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /home/aknight2015/.linuxbrew
HOMEBREW_REPOSITORY: /home/aknight2015/.linuxbrew/Homebrew
HOMEBREW_CELLAR: /home/aknight2015/.linuxbrew/Cellar
HOMEBREW_BROWSER: /usr/bin/garcon-url-handler
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 2
Homebrew Ruby: 2.6.8 => /home/aknight2015/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8_1/bin/ruby
CPU: dual-core 64-bit unknown_0x6_0x5c
Clang: N/A
Git: 2.30.2 => /bin/git
Curl: 7.74.0 => /bin/curl
Kernel: Linux 5.10.110-15808-ge5740beba59b x86_64 GNU/Linux
OS: Debian GNU/Linux 11 (bullseye) (bullseye)
Host glibc: 2.31
/usr/bin/gcc: 10.2.1
/usr/bin/ruby: N/A
glibc: N/A
gcc@5: N/A
xorg: N/A
brew doctor:Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Your Homebrew's prefix is not /home/linuxbrew/.linuxbrew.
Some of Homebrew's bottles (binary packages) can only be used with the default
prefix (/home/linuxbrew/.linuxbrew).
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
unsupported configuration.
